src/atoms/Checkbox/__tests__/Checkbox.web.test.js

```diff
@@ -91,26 +100,113 @@ describe('<Checkbox />', () => {
     });
   });
 
+  describe('checkbox only', () => {
+    it('should render a icon only checkbox', () => {
+      const { container } = renderWithTheme(<Checkbox id={SAMPLE_ID} />);
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('focus', () => {
+    describe('with defaultChecked', () => {
+      it('should render focus styles on checkbox when defaultChecked is false', () => {
+        const { container, getByRole } = renderWithTheme(
+          <Checkbox id={SAMPLE_ID} defaultChecked={false} />,
+        );
+        const checkbox = getByRole('checkbox');
+        checkbox.focus();
+        expect(checkbox).toHaveFocus();
+        expect(container).toMatchSnapshot();
+      });
+
+      it('should render focus styles on checkbox when defaultChecked is true', () => {
+        const { container, getByRole } = renderWithTheme(
+          <Checkbox id={SAMPLE_ID} defaultChecked={true} />,
+        );
+        const checkbox = getByRole('checkbox');
+        checkbox.focus();
+        expect(checkbox).toHaveFocus();
+        expect(container).toMatchSnapshot();
+      });
+    });
+
+    describe('with checked', () => {
+      it('should render focus styles on checkbox when defaultChecked is false', () => {
+        const { container, getByRole } = renderWithTheme(
+          <Checkbox id={SAMPLE_ID} checked={false} />,
+        );
+        const checkbox = getByRole('checkbox');
+        checkbox.focus();
+        expect(checkbox).toHaveFocus();
+        expect(container).toMatchSnapshot();
+      });
+
+      it('should render focus styles on checkbox when defaultChecked is true', () => {
+        const { container, getByRole } = renderWithTheme(
+          <Checkbox id={SAMPLE_ID} checked={true} />,
```

Comment:
The test description says "defaultChecked is true"

Reply:
had updated this things in last push.